R/class-workbook.R

```diff
@@ -302,16 +302,17 @@ wbWorkbook <- R6::R6Class(
     #' @param ... additional arguments
     #' @return a `wbWorkbook` object
     initialize = function(
-      creator          = NULL,
-      title            = NULL,
-      subject          = NULL,
-      category         = NULL,
-      datetime_created = Sys.time(),
-      theme            = NULL,
-      keywords         = NULL,
-      comments         = NULL,
-      manager          = NULL,
-      company          = NULL,
+      creator           = NULL,
+      title             = NULL,
+      subject           = NULL,
+      category          = NULL,
+      datetime_created  = Sys.time(),
+      datetime_modified = Sys.time(),
```

Comment:
Nitpicking and only if we ever want to compare creation and modification date, but in this case, we might want to set this to `NULL` and if `is.null()`, set both to the same `Sys.time()` below. Otherwise we might run into another of these timing issues, where precisely enough time is passed in a `Sys.time()` call so that `creation` and `modification` time differ.